README.md

```diff
@@ -32,7 +32,9 @@ A **[React](https://reactjs.org)**  library for [**Elastic App Search**](https:/
 
 ### Live Demo
 
-Checkout the [live demo of Search UI](https://search-ui-stable.netlify.com).
+Checkout the [live demo of Search UI](https://search-ui-parks-example.glitch.me).
```

Comment:
My 2c for what it's worth - I think I semi prefer the old netlify link just because it doesn't have the loading/spinup time that Glitch does. That being said I'm all for keeping/adding the new "Remix on Glitch" link below!

Reply:
@constancecchen I think I agree